### DISCUSSION TITLE
Welcome to OpenEnglishTranslation Discussions!

### DISCUSSION THREAD

**Discussion by RobH123** [Freely-Given-org/OpenEnglishTranslation--OET#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tmakarios** [comment]
Hi everyone!

I'm Tim, from New Zealand.  I'm now using the OET-RV as my preferred translation for [recordings of weekday Bible readings](https://freespoken.nz/@BibleRadio/about) based on [Gloria Wall's lectionary-based reading plan](http://www.wall.org/~gloria/lect/).  (Treat the _follow_ button at freespoken.nz with suspicion; Castopod's federation seems to be unreliable at present.)  Since two of the three translations I use are works in progress, and since audio editing takes time and effort, I'm not editing the recordings, so sometimes you'll hear me stumble.
